<?xml version="1.0"?>
<launch>

  <node name="twist_sin_pub" 
        pkg="comau_tools" 
        type="twist_sin_pub" 
        output="screen"/>

  <node name="rqt_reconfigure" 
        pkg="rqt_reconfigure" 
        type="rqt_reconfigure" 
        output="screen"/>

</launch>
